packages/rum-core/src/boot/startRum.ts

```diff
@@ -90,6 +94,18 @@ export function startRum(
       getCommonContext,
       reportError
     )
+
+  if (batch) {
+    startUserDataTelemetry(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about starting this component after `startRumBatch` directly?